docs/TuxActions.md

```diff
@@ -0,0 +1,358 @@
+The actionCategory has the following parameters
+
+####<a id="actionCategory-category"></a>Property - `actionCategory.__category__` - type: STRING [#](#actionCategory-category)
+The passed in category property is mapped to this key.  You should not normally need access to this property.
+
+```
+    actionCategory.__category__ === 'todos';
+```
+
+####<a id="actionCategory-source"></a>Property - `actionCategory.__source__` - type: STRING [#](#actionCategory-source)
+The passed in source is mapped here, or it is left undefined if no source is passed in. You should not normally need access to this property.
+
+```
+    actionCategory.__source__ === 'todo_views';
+```
+
+####<a id="actionCategory-register"></a>Method - `actionCategory.register` - type: FUNCTION [#](#actionCategory-register)
+An actionCategory instance is able to invoke [Actions.register](#Actions-register) through its own `register` method.  In which case it will use its own category as the category of actions to register to the Actions object.  See below for more information regarding the [Actions.register](#Actions-register) method.
+
+```
+    actionCategory.register(storeToRegister, actionVerbCallbackMap);
+```
+
+This method is equivalent to
+
+```
+    Actions.register(storeToRegister, {
+      todos: actionVerbCallbackMap
+    });
+```
+
+####<a id="actionCategory-ACTION"></a>Method - `actionCategory.ACTION_VERB` - type: FUNCTION [#](#actionCategory-ACTION)
+Each passed in action verb in the actions array becomes a key within the actionCategory that maps to a method which can be invoked to dispatch the corresponding action.
+
+```
+    actionCategory.add(actionBody); //OR
+    actionCategory.update(actionBody); //ETC
+```
+
+#####<a id="ACTION-actionBody"></a>Parameter - `actionBody` - type: OBJECT - optional [#](#ACTION-actionBody)
+Action body which will be passed in to any listeners on this action.
+
+```
+    var newTodo = {
+      text: 'the actionCategory.add method will invoke the storeToRegister.onAdd '
+      + 'callback and will pass this newTodo object into it'
+    };
+    actionCategory.add(newTodo);
+```
+
+#####<a id="ACTION-type"></a>Property - `ACTION_VERB.type` - type: STRING
+This property is attached to each action verb method and represents the full name of the action (category + '_' + action).  TuxedoJS uses this to find the callbacks that are listening to the dispatched action.  You should not normally need access to this property.
+
+```
+    actionCategory.add.type === 'todos_add';
+```
+
+####<a id="actionCategory-before"></a>Method - `actionCategory.before` - type: FUNCTION [#](#actionCategory-before)
+Accepts an action verb string or array of action verbs and a callback, invokes the callback before dispatching the action when the action method is invoked.  The major use case for the `before` callback is to allow actions to invoke async functions (such as Ajax requests) and then dispatch the results of those requests.
```

Comment:
Should there be an it or then after the first comma? Right now it reads a little bit oddly to me.